examples/cpp-simple/README.md

```diff
@@ -8,9 +8,50 @@ It will
 - Every 10 seconds, write a log saying "Hi! I'm a Game Server"
 - After 60 seconds, call `SDK::Shutdown()` to shut the server down.
 
+## Running
+
+If you want to run the example code without building the container yourself, you can start a single gameserver by running
+```
+kubectl create -f https://raw.githubusercontent.com/googleforgames/agones/master/examples/cpp-simple/gameserver.yaml
+```
+
+The gameserver should exit automatically after about a minute. If it doesn't you can remove it
+by running:
+```bash
+$ GAMESERVER_NAME=$(kubectl get gs -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
+$ kubectl delete gs ${GAMESERVER_NAME}
+```
+
+You can also start a fleet of gameservers by running
+```
+kubectl create -f https://raw.githubusercontent.com/googleforgames/agones/master/examples/cpp-simple/fleet.yaml
+```
+
+The fleet will continuously replace gameservers that exit, so when you are finished you should remove the fleet by running
+```bash
+$ FLEET_NAME=$(kubectl get fleets -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')
```

Comment:
Curious - do we not know the fleet name here?

Reply:
These configs use generate-name and `kubectl create` so we do not know the name. I could change them to have a fixed name, but instead I left them as-is which has been nice for testing b/c it's easy to keep creating new ones (and with image pulling set to always you can see the differences as you change the code).

Reply:
Aah cool. Thanks - was on my phone so hard for me to see.